test/shim_exec.bats

```diff
@@ -90,7 +90,7 @@ teardown() {
   [ "$status" -eq 126 ]
 
   echo "$output" | grep -q "No version set for command dummy" 2>/dev/null
-  echo "$output" | grep -q "you might want to add one of the following in your .tool-versions file" 2>/dev/null
+  echo "$output" | grep -q "Please add one of the following versions in your config file at $PROJECT_DIR/.tool-versions" 2>/dev/null
```

Comment:
There may be some benefit to the more vague `You might want to` wording as setting the version in the `.tool-versions` file is the wrong action in a couple scenarios:

* When you are using legacy version file support and should set or update a legacy version file like `.ruby-version`.
* When you should add the version to your global `.tool-versions` file or another `.tool-versions` file in a parent directory.

Also, should we be directing users to add versions to their `.tool-versions` files or direct them to run `asdf local|global <tool> <version>`? I don't feel strongly either way, but both are valid.

Reply:
>When you are using legacy version file support 

Fair point.

>Also, should we be directing users to add versions to their .tool-versions files or direct them to run asdf local|global <tool> <version>? I don't feel strongly either way, but both are valid.

Since we have commands to manage the editing of the version file, we should probably suggest that. I will update now.

Handling `legacy_version` on appropriate commands should be addressed across the board, perhaps in another issue.

Reply:
`asdf local` creates a `.tool-versions` if there is only the `asdf global` version file available, so perhaps editing `.tool-versions` is okay in this instance 🤔

Reply:
I've taken not of these points and will craft issues to address them at a proj-wide level